src/content/docs/en/basics/astro-pages.mdx

```diff
@@ -96,10 +96,56 @@ Files with the `.html` file extension can be placed in the `src/pages/` director
 
 ## Custom 404 Error Page
 
-For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`.
+For a custom 404 error page, you can create a `404.astro` or `404.md` file in `src/pages`.
 
 This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
 
+## Custom 500 Error Page
+
+For a custom 500 error page, you can create a `500.astro` in `src/pages`.
+
+:::caution
+Once deployed, Astro will only use `500.astro` when using `output: 'hybrid'` for pages that opt-out from prerendering or `output: 'server'`.
+:::
+
+If an error occurs in this file, it will default to the host 500 error page.
```

Comment:
```suggestion
For a custom 500 error page to show for pages in your `hybrid` or `server` project that are [rendered on demand](/en/guides/server-side-rendering/#enable-on-demand-server-rendering), create the file `src/pages/500.astro`. This custom page is not available for prerendered pages.

If an error occurs displaying this page, your host's default 500 error page will be shown to your visitor.
```
Gentle reminder that cautions are reserved for MAJOR OMG ISSUES, not just "how stuff works."  :smile: 

Also, can we be more specific than "an error IN this file?" I tried "displaying this page" because that's maybe what the end user would experience? But what would you say is likely to happen and the best descriptive word here: .... an error occurs RENDERING this file? An error occurs PROCESSING this file? And, is file better than page?

Just for you to consider!

Reply:
yeah I think `rendering` is the right word there! And I forgot banners are banned from docs 😂

Reply:
lol, not *banned*, but USED APPROPRIATELY! Is your house going to burn down? Maybe that's a caution. Are you going to destroy your project/lose data? Maybe that's a caution.

Do you have to "follow regular Astro rules as to how stuff works?" Not a caution!  ;)